File: OCH_Tao/ABC/B/003.js
const [S, T] = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"));
const L = ["a", "t", "c", "o", "d", "e", "r"];
let flag = true;
for (let i = 0; i < S.length; i++) {
  if (S[i] === T[i]) {
    continue;
  } else if (S[i] === "@") {
    if (L.includes(T[i])) {
      continue;
    } else {
      flag = false;
    }
  } else if (T[i] === "@") {
    if (L.includes(S[i])) {
      continue;
    } else {
      flag = false;
    }
  } else {
    flag = false;
  }
}
if (flag) {
  console.log("You can win");
} else {
  console.log("You will lose");
}

File: OCH_Tao/ABC/B/003.py
S = input()
T = input()
L = ["a","t","c","o","d","e","r"]
flag = True

for i in range(len(S)):
  if S[i] == T[i]:
    continue
  elif S[i] == "@":
    if T[i] in L:
      continue
    else:
      flag = False
  elif T[i] == "@":
    if S[i] in L:
      continue
    else:
      flag = False
  else:
    flag = False

if flag:
  print("You can win")
else:
  print("You will lose")